lib/src/models/amba5_bfm/axi/axi5_driver.dart

```diff
@@ -79,6 +79,14 @@ class Axi5ArChannelDriver extends PendingClockedDriver<Axi5ArChannelPacket> {
   /// AXI5 AR Interface.
   final Axi5ArChannelInterface ar;
 
+  /// Capture link utilization/bandwidth over time
+  ///
+  /// Based on the % of cycles in which we want to send a transaction
+  /// and actually can (i.e., credits available).
+  num get linkUtilization => _linkValidAndReadyCount / _linkValidCount;
```

Comment:
why is link utilization a function of the driver?
 
it seems more like it should be a component that consumes the output of a stream?
 
for a generic purpose, people may want to measure starting/ending at specific times, for example, or if its between two hardware elements without a driver